expressEngine/customer/service/customer.js

```diff
@@ -46,6 +47,18 @@ class Service {
         ? { message: success.customerDeleted }
         : { error: error.customerNotFound };
     };
+
+    put = async (body) => {
+      const { id, dataToUpdate } = body;
+      if (!dataToUpdate || Object.entries(dataToUpdate).length === 0) {
+        return { error: error.emptyData };
+      }
+      if (dataToUpdate.email && !modelLib.validateEmail(dataToUpdate.email)) {
+        return { error: error.notUpdated };
+      }
+      const result = await DBService.updateOne(Customer, { _id: id }, dataToUpdate);
+      return result.nModified ? { message: success.customerUpdated } : { error: error.notUpdated };
```

Comment:
@rahuljainss use destructuring for nModified.

Reply:
done